src/guides/v2.4/graphql/mutations/add-products-to-requisition-list.md

```diff
@@ -0,0 +1,102 @@
+---
+group: graphql
+title: addProductsToRequisitionList mutation
+b2b_only: true
+contributor_name: EY
+---
+The `addProductsToRequisitionList` mutation adds products to a requisition list.
+
+This mutation requires a valid [customer authentication token]({{page.baseurl}}/graphql/mutations/generate-customer-token.html).
+
+{:.bs-callout-info}
+Use the [storeConfig query]({{page.baseurl}}/graphql/queries/store-config.html) with the `btob_website_configuration_requisition_list_active` attribute to determine whether requisition lists are supported.
+
+## Syntax
+
+```graphql
+mutation {
+  addProductsToRequisitionList(
+    requisitionListUid: ID!
+    requisitionListItems: [RequisitionListItemsInput!]!
+  ) {
+    AddProductsToRequisitionListOutput
+  }
+}
+```
+
+## Example usage
+
+The following example adds products to a requisition list.
+
+**Request:**
+
+``` graphql
+mutation {
+  addProductsToRequisitionList(
+      requisitionListUid: "1",
+      requisitionListItems: [
+        {
+            sku: "sku"
+            quantity: 1
+            selected_options: ["Y29uZmlndXJhYmxlLzkzLzUz","Y29uZmlndXJhYmxlLzE0NC8xNzE="]
+            entered_options: ["2","3"]
```

Comment:
Delete this line. A product with both selected and entered options would be rare. Keep it a regular configurable product. Plus, the syntax is incorrect.